internal/sequence/sequence.go

```diff
@@ -0,0 +1,131 @@
+package sequence
+
+import (
+	"encoding/binary"
+
+	"cosmossdk.io/errors"
+	"cosmossdk.io/math"
+	storetypes "github.com/cosmos/cosmos-sdk/store/types"
+)
+
+var (
```

Comment:
Do we have any reference for this file?

Reply:
Refer to https://github.com/bnb-chain/greenfield-cosmos-sdk/blob/master/x/group/internal/orm/sequence.go